SimplyWealthProject/Code Draft

```diff
@@ -0,0 +1,114 @@
+##html for signout 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SimplyWealth - Sign out</title>
+    <link rel="stylesheet" href="{% static 'css/homepage.css' %}">
+</head>
+<body>
+    <div class="banner">
+        <header>
+            <div class="logo">
+                <img src="{% static 'img/logo.png' %}" alt="Logo">
+            </div>
+            <nav class="navbar">
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="center-content">
+            <h1>SimplyWealth</h1>
+            <div class="button-container">
+                <a href="{% url 'signup' %}" class="button">Sign Up</a>
+                <a href="{% url 'login_view' %}" class="button">Login</a>
+                <a href="{% url 'logout_view' %}" class="button">Sign out</a>  <!-- Updated button text -->
+            </div>
+        </div>
+    </div>
+</body>
+</html>
+
+
+##python code for signout 
+from django.contrib.auth import logout  # Import the logout function
+def signout_view(request):
+    logout(request)
+    messages.success(request, "You have been successfully signed out.")
+    return redirect('index')
+
+from django.contrib.auth import logout
+from django.shortcuts import redirect
+from django.contrib import messages
+
+def signout_view(request):
+    logout(request)
+    messages.success(request, "You have been successfully signed out.")
+    return redirect(request, "welcome/index.html") # Redirect to the login page
+
+##get the application for search_friends
+from flask import Flask, render_template, request
+import re
+
+app = Flask(__name__)
```

Comment:
@dtyang1  have you tested this with our existing django app? Idk if we can incorporate flask with Django. 

try running your code locally with `python manage.py runserver`